src/Router.jsx

```diff
@@ -1,7 +1,7 @@
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Main from './pages/Main/Main';
-import Login from './pages/Login/Login';
-import SignUp from './pages/Users/SignUp/SignUp';
+import Login from '../src/pages/Login/Login';
+import SignUp from '../src/pages/SignUp/SignUp';
```

Comment:
현재 Router.jsx는 src 폴더 하위에 있고 pages 폴더와 동일 경로인데, 다시 src 폴더 외부로 나갔다가 다시 src 폴더로 들어오는 형태로 작성하셨네요!
이렇게 작성하시면 됩니다.
```suggestion
import Login from './pages/Login/Login';
import SignUp from './pages/SignUp/SignUp';
```